install/traf_checkset_env.sh

```diff
@@ -244,11 +244,14 @@ if [ ! -d ${javapath} ]; then
      exit 1
 fi
 
+source $HOME/.bashrc
```

Comment:
Sourcing ~/.bashrc and looking into this file is not a very good solution, IMHO. There are so many other ways we can set JAVA_HOME. For example: /etc/bashrc, ~/.trafodion, $TRAF_HOME/.sqenvcom.sh.

I would suggest to just check whether JAVA_HOME is set, otherwise duplicate the code in $TRAF_HOME/.sqenvcom.sh that sets JAVA_HOME. If JAVA_HOME is not already set, then you can then add a line 

export JAVA_HOME=...

To the file ~/.trafodion.

Reply:
Before, we must prepare environment step by step as following document: [https://cwiki.apache.org/confluence/display/TRAFODION/Create+Build+Environment]
I only write them in one command. 
Yes. I think this is not a good way too.